package.json

```diff
@@ -1,4 +1,5 @@
 {
+  "packageManager": "pnpm@8.2.0",
```

Comment:
pnpm has a release every week. It will be too hard for me to update version in every `package.json` on every release

Reply:
This is a way to use reproducible version of package manager when corepack is enabled. Like lockfile. Pnpm is quite stable so maybe no need to bump every week. It's not a dependency affecting end user.

Reply:
For example I don't have locally installed yarn and pnpm and only rely on corepack. This field is the only requirement.

Reply:
What will happen when I update `pnpm `to `8.2.1`? Do I need to manually go to every open source project and update `package.json`?

Reply:
I'd say only if lockfile of the project is changed. But if you use corepack then you don't update your global pnpm but update the project to use different version.

If too much burden I'll remove it.

Reply:
Let’s remove it. I understand that it is nice to have, but current implementation of `corepack` is just not useful for real cases.

Reply:
Done